### PR TITLE
Fixes #23242 - Add cli plugins to answer file

### PR DESCRIPTION
**Pull request by chris1984**

### CODE REVIEW

**Comment by chris1984** [comment]
@stbenjam here is the foreman pr

File: config/foreman-answers.yaml [hunk 1]
```diff
@@ -20,6 +20,10 @@ puppet:
   server: true
 foreman::cli: true
 foreman::cli::openscap: false
+foreman::cli::discovery: false
+foreman::cli::tasks: false
+foreman::cli::templates: false
+foreman::cli::remote_execution: false
```
Comment (mmoll) [inline config/foreman-answers.yaml]: could you sort this alphabetically?

**Comment by chris1984** [comment]
@mmoll sorted :)

**Review by ekohl — APPROVED** [review]

**Comment by mmoll** [comment]
merged, thanks @chris1984!

**Pull request merged**